shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
It'd possibly break if you've got an `spk dev` running across an upgrade, but that's going to be true if we switch _id to mean "package id" as well, so I can't think of any additional bad cases that also renaming the collection would trigger.